src/test/java/cloud/eppo/helpers/SubjectAssignment.java

```diff
@@ -0,0 +1,28 @@
+package cloud.eppo.helpers;
+
+import cloud.eppo.ufc.dto.SubjectAttributes;
+
+public class SubjectAssignment {
```

Comment:
Mostly not new code; Largely ported from Android implementation ([source](https://github.com/Eppo-exp/android-sdk/blob/main/eppo/src/androidTest/java/cloud/eppo/android/helpers/SubjectAssignment.java))